src/test/java/org/openmrs/addonindex/scheduled/FetchCoreVersionsListIT.java

```diff
@@ -0,0 +1,65 @@
+package org.openmrs.addonindex.scheduled;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.openmrs.addonindex.domain.artifactory.VersionList;
+import org.openmrs.addonindex.service.artifactory.VersionsService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.util.ReflectionTestUtils;
+import org.springframework.web.client.RestTemplate;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+import static org.openmrs.addonindex.TestUtil.getFileAsString;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+public class FetchCoreVersionsListIT {
+
+    @MockBean
+    private RestTemplateBuilder restTemplateBuilder;
+
+    @MockBean
+    private RestTemplate restTemplate;
+
+    @Autowired
+    private VersionsService versionsService;
+
+    @Autowired
+    private FetchCoreVersionsList fetchCoreVersionsList;
+
+    private VersionList versionList;
+
+    @Before
+    public void setUp() throws Exception {
+        when(restTemplateBuilder.build()).thenReturn(restTemplate);
+
+        ReflectionTestUtils.setField(fetchCoreVersionsList,
```

Comment:
Why do you have to do this? Isn't this already the default behavior that's in application.yml?